agac_torch/configs/point_maze.yaml

```diff
@@ -0,0 +1,38 @@
+"algorithm":
+    "max_steps": 2000000
+    "eval_freq": 10000
+    "num_epochs": 30
+    "num_episodes_eval": 1
+    "train_freq": 1000
+    "discrete": False
+    "gpu": 0
+"logging":
+    "save_models": False
+    "save_models_freq": 100000
+    "use_neptune": True
+    "experiment_name": "AGAC-PointMaze"
+    "neptune_user_name": "user_name"
+    "neptune_project_name": "project_name"
+    "log_grid": False
+"reinforcement_learning":
+    "actor_learning_rate": 0.0003
+    "critic_learning_rate": 0.0003
+    "adversary_learning_rate": 0.00009
+    "layers_dim": [64, 64]
+    "adv_layers_dim": [64, 64]
+    "discount": 0.99
+    "batch_size": 512
+    "lambda_gae": 0.95
+    "intrinsic_reward_coefficient": 0.0004
+    "entropy_coefficient": 0.0
+    "clipping_epsilon": 0.2
+    "value_loss_clip": 1000
+    "value_loss_coeff": 0.25
```

Comment:
Usually `value_loss_coeff` is set to 0.5: have you observed some changes in the performance with `value_loss_coeff=0.25`? (I suspect it does not affect the perf because it is just a constant in the gradient update, but it's fine to keep it as is)

Reply:
It's actually the same thing if I'm not mistaken. The original implem has 
`self.vf_loss = .5 * tf.reduce_mean(tf.maximum(vf_losses1, vf_losses2))` 
and then 
`loss = self.pg_loss  -  [...]  + self.vf_coef * self.vf_loss [...]`.
I didn't add the `0.5` in the torch version and thought I might as well collapse both coefs in the `value_loss_coeff`:
`value_loss = self._value_loss_coeff * torch.max(vloss1, vloss2).mean()`

Reply:
Ok! Makes sense.